common/public/include/SessionServerCommon.h

```diff
@@ -78,13 +78,15 @@ constexpr unsigned int kWrite{2};
 constexpr unsigned int kExecute{1};
 
 /**
- * @brief Socket permissions
+ * @brief Socket permissions and ownership
  */
 struct SocketPermissions
 {
     unsigned int ownerPermissions{kRead | kWrite};
     unsigned int groupPermissions{kRead | kWrite};
     unsigned int otherPermissions{kRead | kWrite};
+    std::string owner;
```

Comment:
Define default values. In my opinion they should be root/root. Unfortunately You missed one functionality, you have to add it - recently Adam has introduced ConfigReader (in server manager). Your new values have to be defined in json config file. Please look at Adam's change to know the details.

Reply:
I was thinking that it would be better to have the default values as empty strings, and then the "SessionManagementServer::initialize" method does not call "chown". This will leave the created socket being owned by the user executing the code (and the group would be their primary group). Perhaps I could add a comment to make this intention clear... or do you still think that I should have root/root as default values?

Reply:
ok, that's fine, please add a comment.